### PR TITLE
Autoscaling

### PR DESCRIPTION
**Pull request by AndyMoore111**
autoscaling actions for aws 

slight change to lib/action.py to deserialise datetime's in responses

### CODE REVIEW

**Comment by Kami** [comment]
Thanks.

We will look shortly when get a chance.

**Review by userlocalhost — CHANGES_REQUESTED** [review]

File: actions/lib/action.py [hunk 1]
```diff
@@ -138,7 +149,10 @@ def do_method(self, module_path, cls, action, **kwargs):
                              'aws_secret_access_key) or region')
 
         resultset = getattr(obj, action)(**kwargs)
-        formatted = self.resultsets.formatter(resultset)
+        formatted = json.loads(
```
[the remark this answers is not in the document]
Comment (AndyMoore111) [reply actions/lib/action.py]: thanks - new update should do this instead

**Comment by userlocalhost** [comment]
Thank you for the correction, LGTM!
If possible, you should write tests for the additional actions.

**Review by userlocalhost — APPROVED** [review]

**Comment by AndyMoore111** [comment]
all these actions are generated - not sure if there's a good way to automate the tests as well but will have a look at options

**Comment by emedvedev** [comment]
Thanks for the contribution! @userlocalhost approved, so merging.

**Pull request merged**